### PR TITLE
Automatically update credits file with latest licence information

### PR DESCRIPTION
**Pull request by Octobob**
This is an automated update of the credits / license attribution file, based on the latest code in https://github.com/OctopusDeploy/OctopusDeploy.

Please review for

- syntactical correctness (ie, markdown looks correct)
- logical correctness (ie, when reading it, it makes sense)
- license conformance (we agree with the license in question and we are able to redistribute it)

This pull request was created by TeamCity project Octofront_AutomatedDocumentationUpdates build 2661644

### CODE REVIEW

File: docs/credits.md [hunk 1]
```diff
@@ -1294,25 +1308,26 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
 | NCrontab.Advanced                                 | jcoutch                                                                   | [https://github.com/jcoutch/NCrontab-Advanced](https://github.com/jcoutch/NCrontab-Advanced)                                                                                                                                | [Apache-2.0](https://github.com/jcoutch/NCrontab-Advanced/blob/master/LICENSE)                                      |
+| NCrontab.Advanced                                 | jcoutch                                                                   | [https://github.com/jcoutch/NCrontab-Advanced](https://github.com/jcoutch/NCrontab-Advanced)                                                                                                                                | [Undefined]()                                                                                                       |
```
Comment (michaelnoonan) [inline docs/credits.md:1318]: This is Apache 2.0 - @N-lson I wonder why it didn't pick this up properly?

File: docs/credits.md [hunk 2]
```diff
@@ -1518,6 +1541,9 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
+| prometheus-net.AspNetCore                         | andrasm,qed-,lakario,sandersaares                                         | [https://github.com/prometheus-net/prometheus-net](https://github.com/prometheus-net/prometheus-net)                                                                                                                        | [MIT](https://www.nuget.org/packages/prometheus-net.AspNetCore/4.2.0/license)                                       |
+| prometheus-net.DotNetRuntime                      | James Luck                                                                | [https://github.com/djluck/prometheus-net.DotNetRuntime](https://github.com/djluck/prometheus-net.DotNetRuntime)                                                                                                            | [Undefined]()                                                                                                       |
```
Comment (michaelnoonan) [inline docs/credits.md:1545]: This one is MIT, again, curious why it didn't work automatically. :) https://github.com/djluck/prometheus-net.DotNetRuntime/blob/master/LICENSE.txt

File: docs/credits.md [hunk 3]
```diff
@@ -1788,12 +1823,14 @@ Octopus Deploy is made possible thanks to the following great third-party produc
...
 | stealthy-require                                  | Nicolai Kamenzky                                                          | [https://github.com/analog-nico/stealthy-require](https://github.com/analog-nico/stealthy-require)                                                                                                                          | [ISC](https://spdx.org/licenses/ISC.html)                                                                           |
+| step-validation                                   | Octopus Deploy                                                            | []()                                                                                                                                                                                                                        | [Undefined]()                                                                                                       |
```
Comment (michaelnoonan) [inline docs/credits.md:1826]: @andrewabest Not sure if you're the best person, but trying anyhow: It appears this library is missing some of the information needed for licensing.

**Review by michaelnoonan — APPROVED** [review]
All of the licenses appear to be OK. Some were not auto-detected. One of our own libraries doesn't appear to have the right information for licensing.

Either way, this change looks good enough to publish, recognising the tooling will automatically update and patch the missing pieces over time.

File: docs/credits.md (hunk 1 above)
Comment (N-lson) [reply docs/credits.md:1318]: This was being picked up twice and we only set the license info if it matched package name + author - fix here https://github.com/OctopusDeploy/DependencyScanner/pull/54

File: docs/credits.md (hunk 2 above)
Comment (N-lson) [reply docs/credits.md:1545]: The package is missing license info, added a special case https://github.com/OctopusDeploy/DependencyScanner/pull/54

**Pull request merged**